tools/build_pak_files.py

```diff
@@ -16,125 +16,125 @@
 
     parser = argparse.ArgumentParser()
     parser.add_argument("--unreal_engine_dir", required=True)
-    parser.add_argument("--contents_dir", required=True)
-    parser.add_argument("--platforms", nargs="*", required=True)
+    parser.add_argument("--perforce_content_dir", required=True)
     parser.add_argument("--output_dir", required=True)
     parser.add_argument("--scene_names")
     args = parser.parse_args()
     
     assert os.path.exists(args.unreal_engine_dir)
+    assert os.path.exists(args.perforce_content_dir)
+
+    unreal_project_dir          = os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "cpp", "unreal_projects", "SpearSim"))
+    uproject                    = os.path.realpath(os.path.join(unreal_project_dir, "SpearSim.uproject"))
+    unreal_project_content_dir  = os.path.realpath(os.path.join(unreal_project_dir, "Content"))
+    perforce_content_shared_dir = os.path.realpath(os.path.join(args.perforce_content_dir, "Shared"))
+    perforce_content_scenes_dir = os.path.realpath(os.path.join(args.perforce_content_dir, "Scenes"))
+    output_dir                  = os.path.realpath(args.output_dir)
+
+    # We do not want to use os.path.realpath(...) here, because that will resolve to the directory inside the user's Perforce workspace.
+    # Instead, we want this path to refer to the symlinked version inside the user's unreal project directory.
+    unreal_project_content_shared_dir = os.path.join(unreal_project_content_dir, "Shared")
+
     if sys.platform == "win32":
+        target_platform   = "Win64"
```

Comment:
This string has to be `Windows`, else cook cmd throws error.

Reply:
Fixed.